contract/tests/data/blocks_headers_685440-687456_mainnet.json

```diff
@@ -0,0 +1,25 @@
+[[{ "block_hash": "000000000000000000016f0484972d135afba541c837d0c07c1530ffeee293cd", "version": 805298180, "prev_block_hash": "00000000000000000006248c28751a176336f5c070f901dc86df190c391d761d", "merkle_root": "534e13aa090e6615a2a6610f49b42ca9caa93f3ce2ca33735ca11444d6705424", "time": 1622337521, "bits": 386752379, "nonce": 1876340370 }],
```

Comment:
I think it will be better to move test data to git-lfs

Reply:
fixed: https://github.com/Near-One/btc-light-client-contract/pull/74/commits/cb968950d99067f1ddf1c3d14e7ad2b614367514